iFixit/checkout-ui-extensions
# Add shipping hint box

Closes https://github.com/iFixit/checkout-ui-extensions/issues/29

Here are the [docs](https://shopify.dev/docs/api/checkout-ui-extensions/2024-01/components) on the components for CR.

## QA

Make sure the UI resembles that of the checkout hint box in shopify-v2.

The shopify-v2 hint box is configurable via theme settings and should show up on the shipping methods page.

<details>
<summary>Screenshot</summary>

![image](https://github.com/iFixit/checkout-ui-extensions/assets/52104630/8a6c2c37-4013-4fc0-ad0f-7d2623a18c9c)

</details>

This implementation is configurable via app block settings, which are normally added in the checkout editor on `ifixit-checkout-dev`.

I think you can QA this one of 2 ways: locally (which might require hardcoding the settings), or in the checkout editor on `ifixit-checkout-test` (this is more realistic but I'm not 100% clear on the steps for that but you can experiment).
